### DISCUSSION TITLE
Increase space between cascading windows?

### DISCUSSION THREAD

**Discussion by lilruffian** [rxhanson/RectanglePro-Community#342] [Q&A]
Is it possible to increase the space between cascading windows? Having them a bit more spread out could help in identifying what's in the background because you'll see more than the top half of window toolbars.

**Comment by rxhanson** [comment]
Here's the terminal command to adjust it:

```
defaults write com.knollsoft.Hookshot cascadeAllDeltaSize -float 29
```

29 is the default. 

Restart the app after. 

It's on my todo to document all of my non-UI defaults configurations for Rectangle Pro.

**Reply by lilruffian** [reply]
🔥🔥🔥

thanks mate
